### DISCUSSION TITLE
Request for an adjustable mod list size

### DISCUSSION THREAD

**Discussion by ghost** [xivdev/Penumbra#224] [Ideas]
Folders are a great way to organize mods, however the deeper you go, the less of the mod title you can see. If the mod title is long enough this could happen with only two folders deep. We can already adjust the Penumbra window size, but the mod list size always stays the same. It would be nice to make that adjustable.

**Comment by Ottermandias** [comment]
![image](https://user-images.githubusercontent.com/70807659/175115490-f0b73c1a-182b-428a-a9a9-7b49a035ee7c.png)
That is an excellent idea, whoever did that would be a genius.

**Comment by ghost** [comment]
I am horribly unobservant. I looked up and down through the settings and just kept glossing over that section. Thank you, thank you! This makes things so much easier!
